docs/DeveloperGuide.md

```diff
@@ -194,6 +194,18 @@ The add applicant feature allows users to add an applicant to the applicant list
 4. If the check is successful, the `AddCommand#execute()` checks if the applicant exists in the applicant list.
 5. If the applicant does not exist, the `Model#addApplicant()` adds the applicant to the applicant list.
 
+#### Design considerations
+
+##### Aspect: Format for adding applicants and interviews
```

Comment:
mhmm this looks good, sheds a light on what we thought through early in the project